### PR TITLE
Tabbed Restautant page using Javascript

### PR DESCRIPTION
**Pull request by raheebwa**
## Highlights
* [x] Bootsrap used for css styling
* [x] All pages are diffeent modules
* [x] Tabs implemented using only Javascript
* [x] Functional programming used mostly

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -85,7 +86,7 @@ module.exports = {
         'max-lines-per-function': 'error',
         'max-nested-callbacks': 'error',
         'max-params': 'error',
-        'max-statements': 'error',
+        // 'max-statements': 'error',
         'max-statements-per-line': 'error',
```
Comment (shloch) [inline .eslintrc.js]: the config in this file should be same as the one provided by microverse [[details here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "prettier": "^1.19.1",
+    "stylelint": "^13.1.0",
+    "stylelint-config-standard": "^20.0.0",
+    "webpack": "^4.41.6",
+    "webpack-cli": "^3.3.11"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (shloch) [inline package.json]: Necessary run common commands like `build`, `development` should be in `package.json` inside `"scripts"`.  [  [see requirements ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png)]

```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: src/modules/home.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    'Our Menu,  Nihil ducimus sapiente ut corporis voluptatibus optio inventore adipisci.Quibusdam perspiciatis neque accusamus nam ? Ratione, quaerat placeat.',
+    'Nihil ducimus sapiente ut corporis voluptatibus optio inventore adipisci.Quibusdam perspiciatis neque accusamus nam ? Ratione, quaerat placeat.'
+];
+//  4. Function to render page 
+const renderHomePage = () => {
+    data.forEach(item => {
+        container.appendChild(paragraph(item));
+    });
+    return container;
+}
```
Comment (shloch) [inline src/modules/home.js]: this is creative, i love it 😊😊

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @raheebwa 

AWESOME JOB...i like your product, it's very professional 😊😊
You organise your code very well
Screen Shot 2020-02-14 at 6 53 07 AM

### [changes required]
Your code works well, but let's polish up some things to make the whole repo more professional and some aspects to meet with requirements 😊. Details below :

- below this page, take a look at this conflicting message, please try to debug that
Screen Shot 2020-02-14 at 6 56 34 AM

More about corrections below :
### Waiting for another review 😊

File: src/modules/home.js (hunk 3 above)
Comment (raheebwa) [reply src/modules/home.js]: Thanks @shlock. Will fix the issues and resubmit

**Comment by raheebwa** [comment]
@shloch I was able to work on the request changes. Thanks for your review

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED 

Hi @raheebwa good job on this one it really looks good, :+1:  and kudos for making all suggested changes :slightly_smiling_face:  **This is approved** 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**